### PR TITLE
node: enable separate locales for electron apps

### PR DESCRIPTION
**Pull request by proletarius101**
It's not quite true that "separate-locales is useless" for Electron apps. We can use [similar technique in Mozilla Firefox](https://searchfox.org/mozilla-central/rev/8a4f55bc09ffc5c25dcb4586c51ae4a9fee77b4c/taskcluster/docker/firefox-flatpak/runme.sh#131-133) with symlinks to achieve effective locale separation.

### CODE REVIEW

File: node/vanilla-quick-start/org.electronjs.ElectronQuickStart.yaml [hunk 1]
```diff
@@ -62,6 +60,9 @@ modules:
       - cp -r electron-quick-start /app
       # Install the wrapper script to start it.
       - install -Dm 755 start-electron-quick-start.sh /app/bin/start-electron-quick-start
+
+      # To allow separate locales
+      - setup-separate-locales ${FLATPAK_DEST}/electron-quick-start/locales
```
Comment (gasinvein) [inline node/vanilla-quick-start/org.electronjs.ElectronQuickStart.yaml:65]: https://github.com/flathub/org.electronjs.Electron2.BaseApp/pull/40#discussion_r1069560427
```suggestion
      - setup-separate-locales ${FLATPAK_DEST}/electron-quick-start/locales
```

**Review by refi64 — APPROVED** [review]
LGTM, thanks!

**Comment by refi64** [comment]
Looks like merging #342 caused some conflicts.

**Comment by refi64** [comment]
Thanks!

**Pull request merged**